### PR TITLE
Standardization logicsource email engagement

### PR DESCRIPTION
**Pull request by izztsfn**
Changes made for the optimization and styling:
1. Removed subqueries and nested CTEs
2. Updated overall formatting
3. Replaced Create statements with Insert
4. Replaced CASE statements that contain only one condition with IF statement instead. Please verify if this change is correct

I have checked the row count and it matches exactly with the existing table. Refer screenshot below:
image

**Note**: once this PR is approved, i will update this script into the scheduler. Thanks!

### CODE REVIEW

File: Clients/Logicsource/Normalization Script/Email Engagement/logicsource_db_email_engagements_log.sql [hunk 1]
```diff
@@ -708,475 +677,439 @@ email_download AS (
...
-        'MQL' AS engagement,
-        description,
-        devicetype,
-           '' AS linkid,
+    FROM form_filled AS activity
+    JOIN `x-marketing.logicsource_hubspot.campaigns` campaign
+      ON activity._campaignID = CAST(campaign.id AS STRING)
+    QUALIFY ROW_NUMBER() OVER(PARTITION BY email, description ORDER BY timestamp DESC) = 1
+  ),
+  contact_list AS (
```
Comment (raaayden) [inline Clients/Logicsource/Normalization Script/Email Engagement/logicsource_db_email_engagements_log.sql:685]: Can you try check if in this CTE has the same data as in prospect_info CTE ? same as the contacts CTE

Comment (izztsfn) [reply Clients/Logicsource/Normalization Script/Email Engagement/logicsource_db_email_engagements_log.sql:685]: Hi @raaayden i have tried comparing the content of all three and they all contain different data, the naming convention may be a bit similar but they're all different
image

**Comment by izztsfn** [comment]
@hafizasyraf72 can briefly check and approve also tq

**Review by raaayden — APPROVED** [review]
I have checked and it looks okay

**Comment by hafizasyraf72** [comment]
Hi @izztsfn, all good from my side including the technical logic

**Pull request merged**